assets/components/radioToggle/radioToggle.jsx

```diff
@@ -46,16 +51,25 @@ export default function RadioToggle(props: PropTypes) {
           id={radioId}
           onChange={() => props.toggleAction(radio.value)}
           checked={radio.value === props.checked}
+          tabIndex="0"
```

Comment:
This takes the radio button out of the tabbing context, this may be an issue for keyboard-only users.